book/tutorials/thermal-ir/index.md

```diff
@@ -0,0 +1,12 @@
+# Thermal Infrared Remote Sensing of Snow
+
+**At the conclusion of this tutorial, you will be able to:**
+- Understand the differences between sources of thermal IR observations, the advantages and disadvantages of each, and types of research questions we can address with those observations
+- Access and visualize airborne and satellite thermal infrared imagery, then scale imagery of different spatial resolutions for comparison
+- Visualize point and raster thermal infrared datasets together, then compute error statistics between point and raster datasets
```

Comment:
This is great. One nice feature of the JupyterBook rendering is using call-out boxes or "admonitions", which you might want to try out https://jupyterbook.org/content/content-blocks.html#notes-warnings-and-other-admonitions 

```{admonition} Learning objectives

- Access and visualize airborne and satellite thermal infrared imagery, then scale imagery of different spatial resolutions for comparison
- Visualize point and raster thermal infrared datasets together, then compute error statistics between point and raster datasets
- etc
```

Reply:
Thanks, I've added a few of these admonition blocks!